src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java

```diff
@@ -30,8 +30,8 @@ private void init() {
     new Navigate(driver).toPage(MercurySubpages.MAIN_PAGE);
   }
```

Comment:
we agreed on calling this type of tests "Navigating". Change the class name and group